XML/Snippet.sublime-syntax

```diff
@@ -0,0 +1,26 @@
+%YAML 1.2
+---
+# See http://www.sublimetext.com/docs/3/syntax.html
+file_extensions:
+  - sublime-snippet
+name: Sublime Text Snippet
+scope: text.xml.sublime-snippet
+contexts:
+  main:
+    - match: '(<)(content)(>)(<!\[CDATA\[)'
```

Comment:
@FichteFoll thanks for the heads up - I've done some testing and you are correct. Also, if more than one CDATA section is used, ST ignores any but the first, so I've marked any others as illegal.